### PR TITLE
@tinacms/react-sidebar

### LINKED ISSUE #954: <div class="SiteWrapper"> included in gatsby build output
## Description

Is the `<div class="SiteWrapper">` supposed still be there when you `gatsby build` your site? If so, is there a way to remove or customize its css properties? Its `position: absolute` is causing some layout issues for me when my site winds up in its final environment.

#### Tinacms Package Versions
├─┬ gatsby-plugin-tinacms@0.1.31
│ └─┬ tinacms@0.18.2
│   ├── @tinacms/alerts@0.1.0
│   ├── @tinacms/core@0.7.3
│   ├─┬ @tinacms/fields@0.9.1
│   ├─┬ @tinacms/form-builder@0.3.0
│   ├─┬ @tinacms/forms@0.3.0
│   ├── @tinacms/icons@0.7.0
│   ├── @tinacms/media@0.1.0
│   ├── @tinacms/react-alerts@0.1.0
│   ├─┬ @tinacms/react-core@0.2.9
│   ├─┬ @tinacms/react-forms@0.1.0
│   ├── @tinacms/react-modals@0.1.0
│   ├── @tinacms/styles@0.4.0
│   └─┬ react-tinacms-editor@0.3.0
├─┬ gatsby-tinacms-git@0.4.5
│ ├─┬ @tinacms/api-git@0.7.0
│ └── @tinacms/git-client@0.6.4
├─┬ gatsby-tinacms-json@0.8.16
├─┬ gatsby-tinacms-remark@0.7.11

### PR DESCRIPTION
**Pull request by ncphillips**
This is it!

Extracted the Sidebar to it's own package.

`tinacms` is now the quick-start package it should be...plus a couple other things. 

- [x] Address the `cms: any` castings

Closes #954

### CODE REVIEW

File: packages/@tinacms/react-sidebar/src/components/Sidebar.tsx [hunk 1]
```diff
@@ -27,47 +27,67 @@ import {
...
 
   React.useEffect(() => {
     if (typeof hidden !== 'undefined') {
-      cms.sidebar.hidden = hidden
+      sidebar.hidden = hidden
     }
   }, [hidden])
 
+  // TODO: Uncomment this
+  // if (sidebar.hidden) return children
```
Comment (ncphillips) [inline packages/@tinacms/react-sidebar/src/components/Sidebar.tsx:58]: @spbyrne I think uncommenting this might solve #954?

Comment (spbyrne) [reply packages/@tinacms/react-sidebar/src/components/Sidebar.tsx:58]: I'll have to test, but it seems that way! That would be a big improvement.

Comment (spbyrne) [reply packages/@tinacms/react-sidebar/src/components/Sidebar.tsx:58]: Works as expected 👍

**Comment by spbyrne** [comment]
closes #954

**Pull request merged**